source/UIAHandler/__init__.py

```diff
@@ -790,13 +986,22 @@ def IUIAutomationNotificationEventHandler_HandleNotificationEvent(
 					f"activityId={activityId}"
 				)
 			return
+		if _isDebug():
+			log.debug(
+				f"Queuing  UIA_notification NVDA event "

```

Comment:
remove f string, fix double space

I'm not sure if this is an issue, but there are many strings that are unnecessarily marked as an f-string (i.e. no special formatting). 
I have seen reviews in the past encouraging f-strings only when required.

```suggestion
				"Queuing UIA_notification NVDA event "
```